### ISSUE TITLE
sol-merger env: node\r: No such file or directory

### ISSUE THREAD

**Issue by xlab** [RyuuGan/sol-merger#31]
Running on macOS, getting this error:

```
$ sol-merger contract.sol
env: node\r: No such file or directory
```

Another relevant issue:
https://github.com/docsifyjs/docsify-cli/issues/78

Fixed by opening `sol-merger` script and change line endings from Windows to Unix:
<img width="442" alt="Screenshot 2020-04-13 at 15 56 08" src="https://user-images.githubusercontent.com/477998/79122042-6706ea80-7d9f-11ea-8a7e-567dbbb5579a.png">

Please update your build env to produce files with unix line endings.

**Comment by RyuuGan** [comment]
Hello @xlab 

I'll try to fix this problem, but it is typescript compiler doing this, because all code is using `\n`, but not Windows style `\r\n`. 

The only thing I'm doing is compiling in windows.

Best regards

**Comment by RyuuGan** [comment]
Next versions with be with `\n` new lines. There is an option to `tsc` to do so: 

```
tsc --newLine lf -p tsconfig.app.json
```

Best regards

**Comment by RyuuGan** [comment]
Should be fixed in `v3.0.0`
